### PR TITLE
Added string translation and gettext support.

### PR DESCRIPTION
**Pull request by theokrammer**
This PR introduces an implementation of the trans filter. I do not completely remove whitespace atm, because restoring the format is complicated and i do not think it is needed. Feel free to correct me if this is otherwise. 
I tested in the BNN environment, but thanks to `config.translator` inside the projects `fractal.config.js` accepting a 
`require('gettext-parser').po.parse(require('fs').readFileSync(...),'UFT-8');` (whatever that is) the path can be adapted to the specific project. If nothing is passed, none of the translation stuff should be used.

Edit: Seems my Settings are broken and the format broke, what format do you guys use?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,26 +0,0 @@
-{
```
Comment (sun) [inline package.json:1]: Why did you delete this file? Please revert.

File: src/tags/index.js [hunk 2]
```diff
@@ -145,13 +145,43 @@ module.exports = function(fractal){
...
-
-                        // Look for plural tag.
+                        fs.appendFile(
+                            require('os').homedir + '/fractal.output',
+                            '\n value: ' + value +
+                            '\n Output: ' + JSON.stringify(token.output),
+                                (err) => {
+                                    if (err) throw err;
+                                    console.log('updated');
+                                });
```
Comment (sun) [inline src/tags/index.js]: Seems like debugging code?

File: src/tags/index.js [hunk 3]
```diff
@@ -200,5 +246,29 @@ module.exports = function(fractal){
             };
         }
     }
-
 };
+
+function split_translation(translated_string, variables) {
```
Comment (sun) [inline src/tags/index.js:245]: Is there a reason why this function is declared outside of the exports?

**Review by sun — CHANGES_REQUESTED** [review]

File: src/tags/index.js (hunk 2 above)
Comment (theokrammer) [reply src/tags/index.js]: As i said, there are certain edgecases where my solution will not work. I assume those cases are very rare, only one of the components in the BNN-templates uses this, although the translation works correctly when viewing the main_site. IIRC it had something to do with whitespace being rendered as `\n` instead of  a single space ` `, what is appearing in the `.po` file, and thus the translation not being done.
Since i did not know if you guys may have some other testing cases i did not wanna remove this, since it was very useful to be able to see all strings to be translated. If you think this is fine i can remove this.

File: src/tags/index.js (hunk 3 above)
Comment (theokrammer) [reply src/tags/index.js:245]: Since it is not needed anywhere else? Why would i export something that is not needed anywhere else?

File: package.json (hunk 1 above)
Comment (theokrammer) [reply package.json:1]: Sorry, readded this.

File: src/tags/index.js (hunk 2 above)
Comment (theokrammer) [reply src/tags/index.js]: @sun did you read my comment?

File: src/tags/index.js (hunk 3 above)
Comment (theokrammer) [reply src/tags/index.js:245]: @sun did you read my reply?

File: src/tags/index.js (hunk 2 above)
Comment (fabianmarz) [reply src/tags/index.js]: @theoshallburn, at least there should be a debug flag or checking the NODE_ENV. Since this is a public repo and the package is not tied to a specific project writing a file might not be desired by some users or use cases. (I added this to my commit I did)

File: src/tags/index.js [hunk 4]
```diff
@@ -141,17 +141,52 @@ module.exports = function(fractal){
...
+                            fs.appendFile(
+                                require('os').homedir + '/fractal.output',
+                                '\n value: ' + value +
+                                '\n Output: ' + JSON.stringify(token.output),
+                                    (err) => {
+                                        if (err) {
+                                            throw err;
+                                        }
+                                    });
+                        }
```
Comment (sun) [inline src/tags/index.js]:
```suggestion
```

File: package.json [hunk 5]
```diff
@@ -17,6 +17,7 @@
     "@frctl/fractal": "latest",
     "callsites": "^3.1.0",
     "deepmerge": "^4.0.0",
+    "gettext-parser": "4.0.4",
```
Comment (sun) [inline package.json]:
```suggestion
    "gettext-parser": "^4.0.4",
```

**Review by sun — APPROVED** [review]
